Modules/_testcapi/watchers.c

```diff
@@ -250,28 +260,31 @@ get_type_modified_events(PyObject *self, PyObject *Py_UNUSED(args))
     return Py_NewRef(g_type_modified_events);
 }
 
+/*[clinic input]
+_testcapi.watch_type
+    watcher_id: int
+    type: object
+    /
+[clinic start generated code]*/
```

Comment:
Please describe what this test code do, it would be a good conversation when we write C API test code.
I did something on https://github.com/python/cpython/pull/104502

Reply:
I deleted the docstring, because as @erlend-aasland said it is quite obvious what this function does. It literally unwatches a type. Do you have any suggestions about the description it needs? :)

Reply:
Ah I now read his comment.

Reply:
I recommend section 1.6 (_"Comments"_) in The Practice of Programming (Kernighan & Pike) for some well written sentences about comments :)